<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto Connect Uni VPN</string>
  </property>
  <widget class="QWidget" name="rootWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="baseVerticalLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="statusHeadlineLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusTextLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Disconnected</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="statusProgressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="dataInputLogLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="switchLogLayout">
          <item>
           <widget class="QPushButton" name="toDataInputButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="toLogButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Log</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="logOutputTextBrowser"/>
        </item>
        <item>
         <layout class="QGridLayout" name="userInputLayout">
          <property name="spacing">
           <number>15</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="uniPasswordLabel">
            <property name="text">
             <string>Uni Password</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="uniIDLineEdit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="uniPasswordLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="uniIDLabel">
            <property name="text">
             <string>Uni-ID</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="uniSeedLabel">
            <property name="text">
             <string>2FA Seed</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="uniSeedLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="connectButtonsLayout">
        <item>
         <widget class="QPushButton" name="disconnectButton">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <addaction name="actionEnglish_EN"/>
     <addaction name="actionDeutsch_DE"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpGuide"/>
    <addaction name="actionHelpGet2FASeed"/>
    <addaction name="actionHelpFAQ"/>
    <addaction name="actionReportBug"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionEnglish_EN">
   <property name="text">
    <string>&amp;English (EN)</string>
   </property>
  </action>
  <action name="actionDeutsch_DE">
   <property name="text">
    <string>&amp;Deutsch (DE)</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>&amp;Info</string>
   </property>
  </action>
  <action name="actionReportBug">
   <property name="text">
    <string>&amp;Report Bug</string>
   </property>
  </action>
  <action name="actionHelpGet2FASeed">
   <property name="text">
    <string>Get &amp;2FA Seed</string>
   </property>
  </action>
  <action name="actionHelpFAQ">
   <property name="text">
    <string>&amp;FAQ</string>
   </property>
  </action>
  <action name="actionHelpGuide">
   <property name="text">
    <string>&amp;Guide</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
